GCRCatalogs/user_config.py

```diff
@@ -0,0 +1,42 @@
+from argparse import ArgumentParser, RawTextHelpFormatter
+from .user_config_mgr import UserConfigManager
+
+if __name__ == "__main__":
+    desc="""Directly manipulate items in user config"""
+
+    parser = ArgumentParser(description=desc, prog='GCRCatalogs.user_config',
```

Comment:
spaces around `=`. `desc` is fine but we also happen to be in the DESC. To avoid confusion, switch to `usage`

```suggestion
    usage = """Directly manipulate items in user config"""

    parser = ArgumentParser(description=usage, prog='GCRCatalogs.user_config',
```

Reply:
`usage` has a different meaning to ArgumentParser so I'd rather not use that.   How about `descrip` or, if that's not acceptalbe, `description`?

Reply:
let's just use `description` then.